<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Create_table</class>
 <widget class="QDialog" name="Create_table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>539</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>880</width>
    <height>539</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>880</width>
    <height>539</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Создание таблицы</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget_4" native="true">
      <property name="styleSheet">
       <string notr="true">background: #1f8ddb</string>
      </property>
      <widget class="QLabel" name="PASSWORD_text_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>131</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft JhengHei</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Название таблицы:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="type_pole">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>320</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #fcfcf2;   border-radius: 10px;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="name_table">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>30</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #fcfcf2;   border-radius: 10px;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="name_pole">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>290</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #fcfcf2;   border-radius: 10px;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="save_create_table">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>260</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid #0f4266;
    border-radius: 10px;
    background: #227cbd;
    padding: 5px;
    }

QPushButton:hover {
    background: #31c3d6;
    }</string>
       </property>
       <property name="text">
        <string>Сохранить таблицу</string>
       </property>
      </widget>
      <widget class="QLabel" name="PASSWORD_text_4">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>30</y>
         <width>91</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft JhengHei</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>База данных:</string>
       </property>
      </widget>
      <widget class="QLabel" name="name_used_database">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>30</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>***Название выбранной базы данных</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>851</width>
         <height>171</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #fcfcf2;   border-radius: 10px;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>290</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Название поля:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>320</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Тип:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>260</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Создание полей таблицы</string>
       </property>
      </widget>
      <widget class="QPushButton" name="save_create_pole">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>440</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid #0f4266;
    border-radius: 10px;
    background: #227cbd;
    padding: 5px;
    }

QPushButton:hover {
    background: #31c3d6;
    }</string>
       </property>
       <property name="text">
        <string>Добавить поле</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="newuser_password_3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>360</y>
         <width>161</width>
         <height>151</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #fcfcf2;   border-radius: 10px;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>360</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Комментарий:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>300</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Storage:</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>300</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Virtual</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>300</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Stored</string>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>330</y>
         <width>311</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>Not Null</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBox_6">
          <property name="text">
           <string>Unsignet</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_5">
          <property name="text">
           <string>Binary</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Primary Key</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>Uniqure</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string>Zero Fill</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QCheckBox" name="checkBox_7">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>430</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Auto increment</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_8">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>430</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
